koa-crud/src/graphql/resolvers/promo-enrollments.ts

```diff
@@ -10,91 +8,62 @@ import {
 } from '../../use-cases/promo-enrollment-requests';
 
 import { NotAllowedError } from '../../custom-errors';
-import { Connection, PromoEnrollmentRequest } from '../../types/index';
 
-const enrollToPromo = async (
-  obj,
-  args: PromoEnrollmentRequest,
-  ctx,
-): Promise<boolean> => {
+import paginate from '../../pagination';
+
+import { PromoEnrollmentRequestDocument } from '../../lib/mongoose/models/promo-enrollment-request';
+import { Connection } from '../../types';
+
+const enrollToPromo = async (obj, args, ctx): Promise<boolean> => {
   if (!ctx.allowed) {
     throw new NotAllowedError('You are not allowed to access this resource');
   }
 
   return enrollToPromoUseCase({
-    id: ctx.userId.data,
-    info: args,
-    source: null,
+    info: {
+      member: ctx.userId.data,
+      promo: args.promo,
+    },
   });
 };
 
-const promoEnrollmentRequests = async (): Promise<
-  Connection<PromoEnrollmentRequest>
-> => {
-  const nodes = await selectAllPromoEnrollmentRequestsUseCase({
-    id: null,
-    info: null,
-    source: null,
-  });
-
-  const edges = R.map((node: PromoEnrollmentRequest) => {
-    return {
-      node,
-      cursor: 'not implemented',
-    };
-  })(nodes);
+const promoEnrollmentRequests = async (
+  obj,
+  args,
```

Comment:
@lafbelmonte  This should have a type, right?
Especially the `obj` and `args`

Reply:
will add types for that sir